### PR TITLE
Provide #[ArrayShape] for some functions

### PR DESCRIPTION
**Pull request by wbars**

### CODE REVIEW

**Comment by orklah** [comment]
Oh, that's cool!

cc @voku

File: Core/Core.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 
 // Start of Core v.5.3.6-13ubuntu3.2
+use JetBrains\PhpStorm\ArrayShape;
```
Comment (mfn) [inline Core/Core.php:4]: 👀  where does that come from 😅

Comment (b1rdex) [reply Core/Core.php:4]: https://github.com/JetBrains/phpstorm-stubs/blob/master/meta/attributes/ArrayShape.php

**Comment by b1rdex** [comment]
Wouldn't it break php < 8.0 runtime?

cc @ondrejmirtes

**Comment by wbars** [comment]
@b1rdex phpstorm-stubs are not meant to be executed by PHP, it serves mostly static analysis purpose.

**Pull request merged**